Instastories.py

```diff
@@ -203,23 +211,53 @@ def nicks_to_ids(usr_list):
     return ids    
 
 
+def numberOfPeople():
+
+    count = int(input("Please insert the number of people to scrape: "))
+    return count
+
+
+
+def logToHTML():
```

Comment:
This function does not render to HTML, but return the log list. Also this is flask_server.py job. Move the function there and change the name.